.github/workflows/build-and-push-image-development.yml

```diff
@@ -0,0 +1,43 @@
+name: Continuous delivery (Development)
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build-and-push-image-development:
+    name: Build and push image development
+    environment: Dev
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+
+      - name: Azure Container Registry login
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DEVELOPMENT_AZURE_ACR_CLIENTID }}
+          password: ${{ secrets.DEVELOPMENT_AZURE_ACR_SECRET }}
+          registry: ${{ secrets.DEVELOPMENT_AZURE_ACR_URL }}
+
+      - name: Prepare tags
+        id: prepare-tags
+        run: |
+          DOCKER_IMAGE=${{ secrets.DEVELOPMENT_AZURE_ACR_URL }}/acatran-app
+          VERSION=latest
+          TAGS="${DOCKER_IMAGE}:${VERSION}"
```

Comment:
This looks good. Though I think we may end up tweaking the format of this tag to match the convention we've been using
e.g. 
![image](https://user-images.githubusercontent.com/109524605/211868821-08aea0a5-d481-4880-9061-849b8dcb0e75.png)

Reply:
Changed to Docker image tag to `dev-YYYY-MM-DD.#` to match the format of the dev release convention